crates/wasmtime/src/runtime/vm/vmcontext.rs

```diff
@@ -263,6 +263,39 @@ mod test_vmglobal_import {
     }
 }
 
+/// The fields compiled code needs to access to utilize a WebAssembly
+/// tag imported from another instance.
+#[derive(Debug, Copy, Clone)]
+#[repr(C)]
+pub struct VMTagImport {
+    /// A pointer to the imported tag description.
+    pub from: VmPtr<VMTagDefinition>,
+}
```

Comment:
Do we actually need a separate `VMTagImport` from `VMTagDefinition`?

Because each tag is immutable (right?) we can inline the original `VMTagDefinition` for imported tags, removing the distinction between imported vs defined tags at the vmctx level. And `VMTagDefinition` happens to be smaller than a pointer on 64-bit architectures, so between that and removing an indirection, it is a double win.

Reply:
The key thing as far as I am aware is that tags are nominal. So we need to equip them with an unique identity, one per module instance.

Reply:
Ah, okay, that is good to know, I didn't realize they were nominal.